src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -62,8 +73,8 @@ struct cbuf_maps {
 };
 
 struct cbuf_info {
-	u32_t cbid;
-	int size;
+	int cbid;
```

Comment:
in the cbuf_create, it should be int.
I afraid we need to cast a lot if we changed it to be unsigned.